e2e/Utils/CaptainTest.cs

```diff
@@ -9,7 +9,7 @@ public class CaptainTest : IDisposable
   {
     AndroidApp = Path.GetFullPath("../../../../example/build/app/outputs/flutter-apk/app-debug.apk"),
     AndroidAppId = "com.instabug.flutter.example",
-    AndroidVersion = "14",
+    AndroidVersion = "13",
```

Comment:
Why do we need to change the Android emulator version in this PR?

Reply:
since circle-ci uses Android 13 ,, so Android version 14 was a mistake :D